<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Angeli by dbitton</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Angeli</h1>
      <h2 class="project-tagline">AnGeLi is a tool for gene set enrichment analysis</h2>
      <a href="https://github.com/dbitton/AnGeLi" class="btn">View on GitHub</a>
      <a href="https://github.com/dbitton/AnGeLi/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/dbitton/AnGeLi/tarball/master" class="btn">Download .tar.gz</a>
    </section>

    <section class="main-content">
      <p>Genome-wide assays and screens typically result in large lists of genes or proteins.  Enrichments of functional or other biological properties within such lists can provide valuable insights and testable hypotheses.  To systematically detect these enrichments can be challenging and time-consuming, because relevant data to compare against query gene lists are spread over many different sources.  We have developed AnGeLi (Analysis of Gene Lists), an intuitive, integrated web-tool for comprehensive and customized interrogation of gene lists from the fission yeast, Schizosaccharomyces pombe.  AnGeLi searches for significant enrichments among multiple qualitative and quantitative information sources, including gene and phenotype ontologies, transcript and translational features, genetic and protein interactions, chromosomal positions, various protein properties such as localization, conservation, domains and molecular weight, and diverse sets of genes that are co-regulated or lead to same phenotypes when mutated.  AnGeLi uses robust statistics which can be tailored to specific needs.  It also provides the option to upload user-defined gene sets to compare against the query list.  Through an integrated data submission form, AnGeLi encourages the community to contribute additional curated gene lists to further increase the usefulness of this resource and to get the most from the ever increasing large-scale experiments.  AnGeLi offers a rigorous yet flexible statistical analysis platform for rich insights into functional enrichments and biological context for query gene lists, thus providing a powerful exploratory tool through which S. pombe researchers can uncover fresh perspectives and unexpected connections from genomic data.  AnGeLi is freely available at: <a href="http://www.bahlerlab.info/AnGeLi">www.bahlerlab.info/AnGeLi</a></p>

      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/dbitton/AnGeLi">Angeli</a> is maintained by <a href="https://github.com/dbitton">dbitton</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
